web/src/beta/components/CheckboxField/index.tsx

```diff
@@ -0,0 +1,57 @@
+import type { FC } from "react";
+
+import { styled, useTheme, PublishTheme } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+export type Props = {
+  publishedTheme?: PublishTheme;
+  onClick?: (value: boolean) => void;
+  checked?: boolean;
+  label: string;
+};
+
+const CheckBoxField: FC<Props> = ({ onClick, checked, label, publishedTheme }) => {
+  const theme = useTheme();
+  return (
+    <Field>
+      <BoxFeild
+        onClick={() => onClick?.(checked !== undefined ? checked : false)}
+        publishedTheme={publishedTheme}>
+        {checked && <CheckMark icon="checkmark" publishedTheme={publishedTheme} />}
+      </BoxFeild>
+      {label && (
+        <Text size="xs" color={publishedTheme?.mainText || theme.main.text}>
+          {label}
+        </Text>
+      )}
+    </Field>
+  );
+};
+
+const Field = styled.div`
```

Comment:
I guess it's better to use `button` here, and it's working like `label > input`.
Then users can change the checkbox state by clicking the label as well.

Reply:
Fix completed.